decisions/wc_class_decisions.txt

```diff
@@ -2095,7 +2095,65 @@ targetted_decisions = {
 	}
 
 	### Hunter ###
+	# Adds tamed beast modifier 
+	tame_beast = {
+		filter = self
+		ai_target_filter = self
+		ai_check_interval = 48
+
+		only_playable = yes
+
+		potential = {
+			is_untouchable_trigger = no
+
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+
+			NOT = { has_character_flag = tame_beast_ongoing_flag }
```

Comment:
The flag isn't cleared.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/85559357-b2a90080-b63a-11ea-908f-25d163c02ac6.png)

</details>

Reply:
Looks like this flag is not needed, so I deleted it